test/e2e/testdata/kustomize-cm-gen/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+kind: Kustomization
+generatorOptions:
```

Comment:
Is this correct way to do this?

Reply:
https://github.com/kubernetes-sigs/kustomize/blob/master/examples/generatorOptions.md